andela/denethor-ah-frontend
# Feature/164396695/create login signup components

#### What does this PR do?
- Implements Login and Sign-up components
- Implements Sign-up functionality

#### How should this be manually tested?
- Clone repo
- Check out to branch `feature/164396695/create login signup components`
- Run application with `npm run dev-server` in the terminal
- Visit `http://localhost:8080/`
- Click on `Get Started` or `Sign Up` buttons.
- Fill the Sign-up form and verify from your account

#### What are the relevant pivotal tracker stories?
- [Create Login and Sign up forms](https://www.pivotaltracker.com/story/show/164396695)
- [Users should be able Signup/Register to Authors Haven](https://www.pivotaltracker.com/story/show/163478757)

#### Screenshots:
![image](https://user-images.githubusercontent.com/25581605/54238448-cfd6c000-4518-11e9-88df-ecde823746cd.png)
![image](https://user-images.githubusercontent.com/25581605/54238522-01e82200-4519-11e9-9167-8d1a9882765c.png)
